tests/app-handover.fg

```diff
@@ -12,31 +12,45 @@ odin.user player3 password1234
 odin.sql.file (module.path.join app-login-test-figure.sql)
 
 # Support only POST
-GET test/app/handover / 501
-GET test/app/handover /app02 501
+GET test/app/handover/mock_user / 501
+GET test/app/handover/mock_user /app02 501
 
 
 # Validate Payload
-POST test/app/handover / {} 501
+POST test/app/handover/mock_user / {} 501
 
-POST test/app/handover / {"token": "mock"} 200
+POST test/app/handover/mock_user / {"token": "mock"} 200
```

Comment:
At the moment the view here is always the string version of the view configuration. What if we allowed a JSON literal at this position as well? Then you could have one-off tests that just have a local configuration.